PRODUCTION_ENVIRONMENT_INSTALLATION.md

```diff
@@ -0,0 +1,323 @@
+# Production environment installation
+
+## Prerequisites
+
+1. We have `root` access to new VPS server with Ubuntu 16.04 preinstalled.
+1. Proper domain (`dev.volontulo.pl`, `rc.volontulo.pl` or `volontulo.pl`) was redirected into new VPS server IP address.
+
+## Update and upgrade
+
+1. Start from updating repositories and upgrading system:
+```
+apt-get update
+apt-get upgrade
+```
+
+## Secrets as environment variables
+
+1. Establish environment variables that will handle our secrets and add them to root's `.bash_profile`:
```

Comment:
Secret key's, not secrets. ;-)

Reply:
I don't believe `Secret key's` is corect gramar, maybe You inteded `secret keys`? BTW for me `secret keys` means subset of more general `secrets` and I would leave it broader here.

Reply:
Secret keys, my bad.